c2rust-transpile/src/translator/assembly.rs

```diff
@@ -444,7 +444,7 @@ fn remove_comments(mut asm: &str) -> String {
     without_comments
 }
 
-/// Detect whether an x86[_64] gcc inline asm string uses Intel or AT&T syntax.
+/// Detect whether an x86(_64) gcc inline asm string uses Intel or AT&T syntax.
```

Comment:
Kind of sad that we have to change this as `[]` was intended to indicate an optional string a la EBNF. Oh well.

Reply:
`()` is more often used to indicate optionality in text, though, so I don't think it's too bad.